accurest-core/src/main/groovy/io/codearte/accurest/builder/SpockMethodBodyBuilder.groovy

```diff
@@ -122,6 +122,17 @@ abstract class SpockMethodBodyBuilder {
 		return trimRepeatedQuotes(json)
 	}
 
+	protected Map<String, Object> getMultipartParameters() {
+		return (Map<String, Object>)request.multipart.serverValue
+	}
+
+	protected String getMultipartParameterLine(Map.Entry<String, Object> parameter) {
+		if (parameter.key == 'file') {
```

Comment:
@wszwech I don't think we should fixate on parameter name (in multipart you can send multiple streamed params with different key), please change this to something like parameter.value instanceof NamedProperty